script.js

```diff
@@ -0,0 +1,11 @@
+myProfilePic
+
+function profileVanish () {
+    let profilePic = document.getElementById('myProfilePic')
+    onclick = profilePic.classList.toggle('show')
```

Comment:
you don't need to set profilePic.classList.toggle('show') to equal anything. you're making essentially a statement. you're saying profilePic toggle show. plus you're not using that onclick variable so it's pointless.